src/trunk.c

```diff
@@ -105,6 +105,19 @@ static const int64 latency_histo_buckets[LATENCYHISTO_SIZE] = {
  * If verbose_logging_enabled is enabled in trunk_config, these functions print
  * to cfg->log_handle.
  */
+void
+trunk_enable_verbose_logging(trunk_handle *spl, platform_log_handle *log_handle)
```

Comment:
Externalized so this can be called directly from unit-test which is not required to go through `--cmd-line-args` parsing.